src/perception/se3.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <string>
+
+#include <tf2_ros/buffer.h>
+#include <tf2_ros/transform_broadcaster.h>
+
+#include <geometry_msgs/Pose.h>
+
+class SE3 : public geometry_msgs::Pose {
+public:
+    [[nodiscard]] static SE3 transform(tf2_ros::Buffer const& buffer, std::string const& fromFrameId, std::string const& toFrameId, SE3 from, uint32_t seq);
```

Comment:
how about something like fromTfTransform() for this one to make it clear its a static factory function as opposed to an operation on the SE3

Reply:
Yuh

Reply:
What about `fromTfTree`?